using-fetch/exercise.js

```diff
@@ -17,10 +17,20 @@ Open index.html in your browser. Every time you refresh the page,
 a different greeting should be displayed in the box.
 */
 
-fetch("*** Write the API address here ***")
+
+ 
+  
+  fetch("https://codeyourfuture.herokuapp.com/api/greetings")
```

Comment:
This is a reasonable attempt, considering that the API unfortunately wasn't available when working on this!

There is another API that is currently working, which does the same thing - note that it responds with an object, so we can use object destructuring to access the greeting:

```js
fetch("https://www.greetingsapi.com/random")
  .then(function (response) {
    if (!response.ok) {
      throw new Error("Network response was not ok");
    }
    return response.json();
  })
  .then(function ({ greeting }) {
    document.getElementById("greeting-text").innerText = greeting;
  })
  .catch(function (error) {
    console.error("Something went wrong:", error);
  });
```